### ISSUE TITLE
[PRC] How may I be of help?

### ISSUE THREAD

**Issue by JJ** [fayland/perl-javascript-beautifier#6]
I have been assigned this distribution. How could I help with it? Maybe work on outstanding issues?

**Comment by fayland** [comment]
Yes sure. I didn't maintain it anymore and if you want, I can assign the module owner to your PAUSE id.

Thanks

**Comment by JJ** [comment]
Let me mull over it for a while and check out the code, maybe do some of
the issues, if that's OK.

**Comment by fayland** [comment]
sure, do whatever you like. thanks.
